pkgs/misc/emulators/dolphin-emu/master.nix

```
@@ -50,14 +50,14 @@ in stdenv.mkDerivation rec {
     "-DDOLPHIN_WC_REVISION=${src.rev}"
     "-DDOLPHIN_WC_DESCRIBE=${version}"
     "-DDOLPHIN_WC_BRANCH=master"
-  ] ++ stdenv.lib.optionals stdenv.isDarwin [
+  ] ++ lib.optionals stdenv.isDarwin [
     "-DOSX_USE_DEFAULT_SEARCH_PATH=True"
   ];
 
   # - Allow Dolphin to use nix-provided libraries instead of building them
   preConfigure = ''
```

```suggestion
  postPatch = ''
```

this is technically patching the code